upgrade guard explanation (#609)
apache/cloudstack-documentation: source/upgrading/index.rst

either prior upgrade or during fresh installation, ACS will handle the Template registration automatically, by mounting the secondary store onto the
management server, copying the respective Templates to the store and then creating the `template.properties` file.

From ACS 4.20.3 and 4.22.1 onwards A starting MS that detects an
upgrade is needed but also sees mshost entries for MSses that are up,

> will abort. If as an operator starting an MS you get the error

Reviewer: ```suggestion
From ACS 4.20.3 and 4.22.1 onwards, a starting Management Server that detects an
upgrade is needed will abort if it also finds mshost entries for other Management Servers that are up.
If as an operator starting a Management Server you get the error
```

`Database upgrade is required but the management server is running in
a clustered environment.` during upgrade, check if you have stopped

> all MSses in your environment and retry. If is still fails it might be

Reviewer: ```suggestion
all other Management Servers in your environment and retry. If it still fails, it might be
```

> that an MS has stopped un-gracefully and you need to fix the DB-entry

Reviewer: ```suggestion
that a Management Server has stopped un-gracefully and you need to fix the DB-entry
```

for it. For example, make sure all MSses are stopped by checking the
hosts in the output of `select name from mshost where state <> ‘Down’`

> and then, when all is safe, run `update mshost set state = ‘Down'`

Reviewer: ```suggestion
for it. For example, make sure all Management Servers are stopped by checking the
hosts in the output of `select name from mshost where state <> ‘Down’`
and then, when all is safe, run `update mshost set state = ‘Down'`
```
